src/App.tsx

```diff
@@ -15,36 +17,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABETICALLY = 'alphabetically',
+  BY_LENGTH = 'by_length',
+  REVERSE = 'reverse',
+  DEFAULT = 'default',
+}
+
+const getPreparedGoods = (
+  goods: string[],
+  currentField: SortType,
+  isReverse: boolean,
```

Comment:
You don`t need isReverce. You can use currentField and check SortType.REVERSE as in other cases.